docs/abi-spec.rst

```diff
@@ -16,7 +16,7 @@ as described in this specification. The encoding is not self describing and thus
 We assume that the interface functions of a contract are strongly typed, known at compilation time and static.
 We assume that all contracts will have the interface definitions of any contracts they call available at compile-time.
 
-This specification does not address contracts whose interface is dynamic or otherwise known only at run-time.
+This specification does not address contracts whose interface is dynamic or otherwise known only at run-time. Also, the ABI specification for libraries is :ref:`slightly different <library-selectors>`.
```

Comment:
```suggestion
This specification does not address contracts whose interface is dynamic or otherwise known only at run-time.

.. note::
   The ABI specification for libraries is :ref:`slightly different <library-selectors>`.
```

I think a note would work better here - i.e. make it more visible.

Reply:
IMO normal text with a link draws more attention and goes better with the reading flow. I personally sometimes miss notes and they feel a little out of context. But I'm fine with a note

Reply:
Given that the changes in the other files are more or less identical, it's perfectly fine to merge as is.